man/cc-test-reporter-format-coverage.1.md

```diff
@@ -33,8 +33,7 @@ tests were run. Defaults to current working directory.
 
 ## COVERAGE_FILE
 
-Path where the coverage file is located. It requires the flag `--input-type`
-to be set.
+(Optional) Explicit path where the coverage file is located. If not passed, the path to the file is inferred based on standard some coverage tool locations. If specifying an explicit COVERAGE_FILE, an `--input-type` must also be specified.
```

Comment:
`Explicit` feels redundant. How about *Path to the coverage file to process.*?

Stating that if you pass a `COVERAGE_FILE` then `--input-type` is required should not be done here, it should result in an update to the header to show the two forms that are now valid. *Something* like:

```
cc-test-reporter-format-coverage [--output=\<path>] [--prefix=<path>]

cc-test-reporter-format-coverage [--output=\<path>] [--prefix=<path>] --input-type=<coverage type> COVERAGE_FILE
```

Doing this means you can probably drop the last sentence from the `--input-type` docs as well.

Also, I'm guessing that fact is not actually enforced by the argument parser at this time, eh?